src/Microsoft.Graph.Core/Exceptions/ClientException.cs

```diff
@@ -9,14 +9,14 @@ namespace Microsoft.Graph
     /// <summary>
     /// Graph client exception.
     /// </summary>
-    public class ClientException : ServiceException
+    public class ClientException : Exception
```

Comment:
should we have this one also inherit api exception so consumers can implement generic try catch clauses?